packages/dds/tree/src/simple-tree/api/tree.ts

```diff
@@ -67,6 +67,13 @@ export interface ViewableTree {
 	viewWith<TRoot extends ImplicitFieldSchema>(
 		config: TreeViewConfiguration<TRoot>,
 	): TreeView<TRoot>;
+
+	// TODO:
+	// Add stored key versions of Tree.exportVerbose, Tree.exportConcise and Tree.exportCompressed here so tree content can be accessed without a view schema.
```

Comment:
link here for context would be good I think

Reply:
I have moved these comments to the interface doc comment so they can use the link syntax. They are in privateRemarks though, so I don't think there is any way to click them, and they don't render as links in intelisense.